rasa/nlu/extractors/crf_entity_extractor.py

```diff
@@ -121,20 +83,70 @@ def required_components(cls) -> List[Type[Component]]:
         "digit": lambda crf_token: crf_token.text.isdigit(),
         "pattern": lambda crf_token: crf_token.pattern,
         "text_dense_features": (
-            lambda crf_token: CRFEntityExtractor._convert_dense_features_for_crfsuite(
+            lambda crf_token: CRFEntityExtractorGraphComponent._convert_dense_features_for_crfsuite(
```

Comment:
I'm unsure how to modify this as to pass the code quality test 😬

Reply:
haha 😁 I'd do it like [here](https://github.com/RasaHQ/rasa/blob/562eef087924e5fe796151689e080b7f65156b06/rasa/shared/importers/autoconfig.py#L112) and add a `# noqa: E501`